build-support/src/main/kotlin/BuildSupport.kt

```diff
@@ -0,0 +1,8 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class BuildSupport : Plugin<Project> {
```

Comment:
I couldn’t figure out how to use functions from this module in my `build.gradle.kts` files without also implementing the plugin interface.

Reply:
Super easy in Groovy Gradle 😁

Reply:
TRUE